api/src/main/java/net/md_5/bungee/api/boss/BossBarBuilder.java

```diff
@@ -0,0 +1,203 @@
+     *
+     * @param color the color you wish to set
+     * @return this BossBarBuilder for chaining
+     */
+    public BossBarBuilder color(BossBarColor color)
+    {
+        this.color = color;
+        return this;
+    }
+
+    /**
+     * Set the current boss bar's division (style)
+     *
+     * @param division the division (style) you wish to set
+     * @return this BossBarBuilder for chaining
+     */
+    public BossBarBuilder division(BossBarDivision division)
+    {
+        this.division = division;
+        return this;
+    }
+
+    /**
+     * Set the current boss bar's health (progress). The number specified should be
+     * between 0 and 1 including.
+     *
+     * @param health the health (progress) you wish to set.
+     * @return this BossBarBuilder for chaining
+     */
+    public BossBarBuilder health(float health)
+    {
+        this.health = health;
+        return this;
+    }
+
+    /**
+     * Adds a player to the boss bar.
+     *
+     * @param player the player you wish to add
+     * @return this BossBarBuilder for chaining
+     */
+    public BossBarBuilder player(ProxiedPlayer player)
+    {
+        Preconditions.checkNotNull( player, "player" );
+        players.add( player );
+        return this;
+    }
+
+    /**
+     * Adds the specified players to the boss bar
+     *
+     * @param players the players you wish to add
+     * @return this BossBarBuilder for chaining
+     */
+    public BossBarBuilder players(ProxiedPlayer... players)
+    {
+        Preconditions.checkNotNull( players, "players" );
+        this.players.addAll( Arrays.asList( players ) );
+        return this;
+    }
+
+    /**
+     * Adds the specified players to the boss bar
+     *
+     * @param players the players you wish to add
+     * @return this BossBarBuilder for chaining
+     */
+    public BossBarBuilder players(Iterable<ProxiedPlayer> players)
+    {
+        for ( ProxiedPlayer player : players )
+        {
+            player( player );
+        }
+        return this;
+    }
+
+    /**
+     * Adds the specified flag(s) to the boss bar.
+     *
+     * @param flags the flag(s) you wish to add
+     * @return this BossBarBuilder for chaining
+     */
+    public BossBarBuilder flags(BossBarFlag... flags)
+    {
+        this.flags = flags;
+        return this;
+    }
+
+    /**
+     * Builds every set boss bar component into a {@link BossBar}
+     *
+     * @return boss bar
+     */
+    public BossBar build()
+    {
+        BossBar bossBar = ProxyServer.getInstance().createBossBar( title, color, division, health );
+        if ( flags.length != 0 )
+        {
+            bossBar.addFlags( flags );
+        }
+        if ( players.size() != 0 )
+        {
+            bossBar.addPlayers( players );
+        }
+        return bossBar;
+    }
+
+    //
+    private void initDefault()
+    {
+        this.title = new ComponentBuilder( "Title not specified" ).create();
```

Comment:
Use this consistently

Reply:
All these init methods are unneeded, just put them into the constructor and call other constructors.

Reply:
> Use this consistently

Wdym?